hazelcast-sql/src/test/java/com/hazelcast/sql/SqlErrorClientTest.java

```diff
@@ -83,37 +82,7 @@ protected ClientConfig clientConfig() {
 
     @Test
     public void testTimeout_execute() {
-        checkTimeout(true);
-    }
-
-    @Test
-    public void testTimeout_fetch() {
-        checkTimeout(true, SqlStatement.DEFAULT_CURSOR_BUFFER_SIZE * 4);
-    }
-
-    @Test
-    public void testExecutionError_fromFirstMember() {
-        checkExecutionError(true, true);
-    }
-
-    @Test
-    public void testExecutionError_fromSecondMember() {
-        checkExecutionError(true, false);
-    }
-
-    @Test
-    public void testMapMigration() {
-        checkMapMigration(true);
-    }
-
-    @Test
-    public void testMapDestroy_firstMember() {
```

Comment:
What happens now when map is destroyed during a query? Don't we want to test it?

Reply:
No, we don't. Citing @viliam-durina 

> Our guarantee is that we'll deliver all entries that weren't added or removed during the scan. For added/modified entries, it's ok if we include them and also if we don't. Map destroy is equivalent to deleting all entries. We keep our guarantee if we stop the query successfully with only a part of the original map contents, which is what i guess happens now. That doesn't contradict the guarantee.
So I'd say we don't have to implement an extra check.
Another issue is that by scanning the map, we might re-create the proxy. But I think that's also OK - the map lifecycle in HZ isn't strict, every access operation creates the map. If you want to fully destroy the map, stop all queries and processes accessing it and then destroy.